### DISCUSSION TITLE
Update the Next.js version of the app directory

### DISCUSSION THREAD

**Discussion by JuanQuiro** [yazaldefilimonepinto/clean-architecture-nextjs-tdd-boilerplate#3] [Ideas]
Currently the new version of Next.js13 is released.

**Comment by yazaldefilimone** [comment]
hei @JuanQuiro, Thank you!

I'm aware of that, and I'm a little busy with some stuff, but if you want to submit a pr, I'm happy to merge it!
if you can't, I'll try to take the time to update
